modules/rest-api/pages/rest-logs-get.adoc

```diff
@@ -40,6 +40,18 @@ For a complete list of log files, see xref:manage:manage-logging/manage-logging.
 
 If no `log-name` argument is specified, the default value is `debug`; whereby the contents of the `debug.log` file are displayed.
 
+== Required Privileges
+
+You must have one of the following roles to call this endpoint:
+
+ * Full Admin 
```

Comment:
Is there a reason these aren't the same as above? e.g.

* xref:learn:security/roles.adoc#views-reader[Views Reader]